### PR TITLE
Add indentation & merge the status page

### PR DESCRIPTION
**Pull request by kerrtravers**
### Why are you making this request?
To add indentation and merge the status page to the main repo.

### What issue does this reference/resolve?
#5 

### Other
31343862-ceba76fc-ad08-11e7-8116-25b78d5826ef
Was this what you meant by indentation? (I fixed the line above too , It's just not in this screenshot)

### CODE REVIEW

**Comment by Kenny2github** [comment]
That does make it flow more smoothly - I assume that “and five new files: a, b, c...” is still line-broken because the line is too long?

**Review by Kenny2github — COMMENTED** [review]
What’s with all the extra files? Is that for ie8?

**Comment by kerrtravers** [comment]
Not too sure as I didn't make the theme, But I'll try and clear some out.

**Comment by kerrtravers** [comment]
@Kenny2github Deleted some unneeded files and dropped ie8 👍

File: Status/assets/css/font-awesome.min.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
...
..."\f29e"}.fa-volume-control-phone:before{content:"\f2a0"}.fa-braille:before{content:"\f2a1"}.fa-assistive-listening-systems:before{content:"\f2a2"}.fa-asl-interpreting:before,.fa-american-sign-language-interpreting:before{content:"\f2a3"}.fa-deafness:before,.fa-hard-of-hearing:before,.fa-deaf:before{content:"\f2a4"}.fa-glide:before{content:"\f2a5"}.fa-glide-g:before{content:"\f2a6"}.fa-signing:before,.fa-sign-language:before{content:"\f2a7"}.fa-low-vision:before{content:"\f2a8"}.fa-viadeo:before{content:"\f2a9"}.fa-viadeo-square:before{content:"\f2aa"}.fa-snapchat:before{content:"\f2ab"}.fa-snapchat-ghost:before{content:"\f2ac"}.fa-snapchat-square:before{content:"\f2ad"}.fa-pied-piper:before{content:"\f2ae"}.fa-first-order:before{content:"\f2b0"}.fa-yoast:before{content:"\f2b1"}.fa-themeisle:before{content:"\f2b2"}.fa-google-plus-circle:before,.fa-google-plus-official:before{content:"\f2b3"}.fa-fa:before,.fa-font-awesome:before{content:"\f2b4"}.sr-only{position:absolute;width:1px;height:1px;padding:0;margin:-1px;overflow:hidden;clip:rect(0, 0, 0, 0);border:0}.sr-only-focusable:active,.sr-only-focusable:focus{position:static;width:auto;height:auto;margin:0;overflow:visible;clip:auto}
```
Comment (Kenny2github) [inline Status/assets/css/font-awesome.min.css]: Could you indent this? This is basically unreadable...

File: Status/assets/js/respond.min.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
...("/"));var h=function(a){return a.replace(c.regex.urls,"$1"+b+"$2$3")},i=!f&&d;b.length&&(b+="/"),i&&(f=1);for(var j=0;f>j;j++){var k,l,o,p;i?(k=d,n.push(h(a))):(k=e[j].match(c.regex.findStyles)&&RegExp.$1,n.push(RegExp.$2&&h(RegExp.$2))),o=k.split(","),p=o.length;for(var q=0;p>q;q++)l=o[q],g(l)||m.push({media:l.split("(")[0].match(c.regex.only)&&RegExp.$2||"all",rules:n.length-1,hasquery:l.indexOf("(")>-1,minw:l.match(c.regex.minw)&&parseFloat(RegExp.$1)+(RegExp.$2||""),maxw:l.match(c.regex.maxw)&&parseFloat(RegExp.$1)+(RegExp.$2||"")})}v()},x=function(){if(d.length){var b=d.shift();f(b.href,function(c){w(c,b.href,b.media),p[b.href]=!0,a.setTimeout(function(){x()},0)})}},y=function(){for(var b=0;b<t.length;b++){var c=t[b],e=c.href,f=c.media,g=c.rel&&"stylesheet"===c.rel.toLowerCase();e&&g&&!p[e]&&(c.styleSheet&&c.styleSheet.rawCssText?(w(c.styleSheet.rawCssText,e,f),p[e]=!0):(!/^([a-zA-Z:]*\/\/)/.test(e)&&!s||e.replace(RegExp.$1,"").split("/")[0]===a.location.host)&&("//"===e.substring(0,2)&&(e=a.location.protocol+e),d.push({href:e,media:f})))}x()};y(),c.update=y,c.getEmValue=u,a.addEventListener?a.addEventListener("resize",b,!1):a.attachEvent&&a.attachEvent("onresize",b)}}(this);
```
Comment (Kenny2github) [inline Status/assets/js/respond.min.js]: Could you indent this too?

File: index.html [hunk 3]
```diff
@@ -22,8 +22,11 @@
 	<li>Download <a href="https://smelt-modding.github.io/smelt/setup.bat"><code>setup.bat</code></a>.</li>
 	<li>Double-click it.</li>
 	<li>You're done! There will be:<ul>
-	<li>two new folders: <code>extension</code> and <code>logs</code></li>
-	<li>five new files: <code>sbpm.bat</code>, <code>install.bat</code>, <code>uninstall.bat</code>, <code>update.cmd</code>, and <code>reinstall.bat</code></li>
+	<li>two new folders: 
+	<br/><code>extension</code> and <code>logs</code></li>
```
Comment (Kenny2github) [inline index.html:26]: well done here :D

**Review by Kenny2github — CHANGES_REQUESTED** [review]
Thanks for the changes! It looks like there are a couple of files that need indentation. If you can't get to them immediately, don't worry - they look pretty large. Work on them when you have time.

File: index.html (hunk 3 above)
Comment (kerrtravers) [reply index.html:26]: Thanks :P

File: Status/assets/js/respond.min.js (hunk 2 above)
Comment (kerrtravers) [reply Status/assets/js/respond.min.js]: Yep

File: Status/assets/css/font-awesome.min.css (hunk 1 above)
Comment (kerrtravers) [reply Status/assets/css/font-awesome.min.css]: Sure!

**Comment by kerrtravers** [comment]
I tried but realized that It's **extremely** hard to indent code you didn't write and know nothing about, Any Ideas?

I hope to move the status page over to the main style soon but a theme was the best option right now, I've also added the header image to the 404 page. 😄

**Comment by Kenny2github** [comment]
Don’t worry, I’ll work on it myself then :)
Besides that, I think this is good to go!

**Review by Kenny2github — APPROVED** [review]

**Comment by kerrtravers** [comment]
Great, Thanks!

**Pull request merged**